apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -651,4 +652,36 @@ export class ApplicationController {
 
     return application
   }
+
+  @Put('application/:id/presignedUrl')
```

Comment:
Looks like we have some inconsistency over the names, we either use `applications/:id` or `application/:id`. We will need to fix that in another PR

Reply:
Nice catch, we can fix this in this PR if you want. Did not notice that you had always `applications/:id`. Why is it plural? To me it makes sense that you are working with one application per that id. 
Anyways don't mind fixing this in this PR if you want 😊

Reply:
If you are up for it, go for it 👍  Yeah I also prefer application/:id for a single application endpoint

Reply:
One question about this. How will this work in the buildpipe and on local machines? 
Everybody will have to clean their cache for the autogenareted files for this to update since we are just changing the name of the endpoint, right?

Reply:
The CI is not using any cache, so it will regenerate all files. Locally, it's common that if you get a type issue to re-run `yarn schemas --skip-cache`

Reply:
Fixed